tests/test_sarif_file_utils.py

```diff
@@ -0,0 +1,31 @@
+from sarif import sarif_file_utils
+
+
+def test_combine_code_and_description():
```

Comment:
Personal preference perhaps, but I think this would be more readable if split into multiple test methods or if there was at least whitespace between the scenarios.

Reply:
Good point, I've split this into three tests